movement/io/load_poses.py

```diff
@@ -69,7 +69,7 @@ def _parse_dlc_csv_to_dataframe(file_path: Path) -> pd.DataFrame:
         DataFrame containing the DLC poses, with multi-index columns.
     """
 
-    possible_level_names = ["scorer", "bodyparts", "coords", "individual"]
+    possible_level_names = ["scorer", "bodyparts", "coords", "individuals"]
```

Comment:
Actually, while we are at it. let's also change the order of the terms, so that they much the DLC file.
```suggestion
    possible_level_names = ["scorer", "individuals", "bodyparts", "coords"]
```